libraries/helpers.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+# frozen_string_literal: true
```

Comment:
This seems unnecessary. Not sure how this line got duplicated, but it is harmless